### ISSUE TITLE
Inventory GUI tabs issue with NEI hidden

### ISSUE THREAD

**Issue by crunkazcanbe** [micdoodle8/Galacticraft#1505]
As the title says here is a picture https://imgur.com/hH6DhqJ

When I remove this mod it is fixed .

**Comment by radfast** [comment]
1.  Which Minecraft and Galacticraft versions?
2.  Which other mods do you have installed?  (For example, CustomNPCs)
3.  Please try out with installing Tinker's Construct mod, let us know if you see the same issue with that.

**Comment by crunkazcanbe** [comment]
Using latest forge 1307 with this being only mod installed also tried it with tinkers by itself and does the same thing . Is there a way to fix it?

**Comment by Dream-Master** [comment]
Why using 1307? The latest Recommended version is 10.13.2.1291.
use this and try again.

**Comment by radfast** [comment]
Tinkers and Galacticraft use the same system of tabs (actually we use their system, for 100% compatibility).

Sounds like Dream-Master may have a solution, thank you for that.

If there's still a problem then this will have to be reported as an issue over at Tinker's Construct Github, as it's their code which is causing this.

**Comment by crunkazcanbe** [comment]
With the newest fast raft mod it is suggested to use 1307 because there is a fix in there that fixes structure gen that i was getting a crash with the recommended version of forge. What i was told is the only Change in 1307 was dealing with structure gen.

**Comment by crunkazcanbe** [comment]
Edit: i also tried 1291 same issue.

**Comment by Dream-Master** [comment]
I am using Galacticraft and tinkers together. I never have a problem with the Gui.
Wich version of tinkers and Galacticraft you are using ?

**Comment by Dream-Master** [comment]
You using Baubles and Thaumcraft I think. (see the ring Symbol in your Pic) This Mod adding a Gui too. (btw I have Thaumcraft too) What Version do you use?

**Comment by crunkazcanbe** [comment]
Yeah I know I have baubles , but I was kinda not counting the mods in the 1.7.10 folder. lol

**Comment by Dream-Master** [comment]
Why you using a dev build form tinkers. Better using 1.83b and try it again.

**Comment by crunkazcanbe** [comment]
Same thing with that version . Do you want me to post a pix of my mods folder and ingame ?

**Comment by Dream-Master** [comment]
Yea post your Mods maybe we can find the bug.

**Comment by radfast** [comment]
> Yeah I know I have baubles , but I was kinda not counting the mods in the 1.7.10 folder. lol

ffs.  Test it with only Galacticraft and tell us if there is the problem still.  If the problem goes when you remove all the other mods, then add them back in in batches until you find the one which is conflicting.  Start with Baubles.

**Comment by crunkazcanbe** [comment]
Damn I hate having to prove , but here it is a picture of my mods folder https://imgur.com/qUuRX2r in game mod list https://imgur.com/OV8a4wK and F3 with the inventory up showing the issue https://imgur.com/7DOyLtv

Hope this helps BTW I am not the only one with this prob ;because I just watched a video on Youtube with someone playing with this mod a while back , and having the same issue .

**Comment by radfast** [comment]
Do you have the same issue when using __normal__ release builds not __dev__ builds?

Please understand that we very much appreciate the issue report (and if you are seeing it then probably other people are too who have not reported it).  But we cannot start to fix an issue until we can reproduce it on our own PCs, and right now I cannot reproduce it... therefore I need precise information on which builds you are using, and asking about other mods is also a part of that.

**Comment by crunkazcanbe** [comment]
Well it does it with normal release versions of tinkers installed by itself . Ill try a normal release version of galacticraft when i get off work, but if im not mistaken the release versions of GC do it to. 

So your running the same forge 1291 not in a dev environment , and not getting the same issue im getting? That's weird how just my PC could do this.

**Comment by crunkazcanbe** [comment]
I am truly sorry if i sounded like a jerk . I really hope we can figure this out. Thanks

-crunk

**Comment by radfast** [comment]
other questions:  what is your screen resolution, are you playing Minecraft full screen or windowed, and what is your GUI text size (normal, large, etc) in the Minecraft options settings?   Maybe post a screenshot of your video settings page in the Minecraft options.

**Comment by crunkazcanbe** [comment]
GUI size is normal and screen res is 2560x1600 but it also does it on my wife's pc with a res of 1920x1200

**Comment by crunkazcanbe** [comment]
I will be happy to post a screenshot when i get off work. :)

**Comment by crunkazcanbe** [comment]
I believe the aspect ratio on both screens is 16:10

**Comment by radfast** [comment]
OK, I have reproduced this - it happens in all Minecraft 1.7.10 Forge versions whenever the player does __not__ have NEI installed, and there is a potion effect.  I guess the 95% majority of players play Galacticraft with NEI (it's certainly a recommended pairing).

Thanks for all the info and patience with the questions.  We should be able to fix this fairly quickly now, and I'll post the fix over at Tinker's Construct as well.

**Comment by crunkazcanbe** [comment]
I do use NEI , and it still happens with NEI installed , but only when you hide NEI with the o key. I tend to keep NEI hidden most of the time when not using it .

**Comment by Cracks777** [comment]
yes i can confirm that it does happen with nei installed if hidden

**Comment by micdoodle8** [comment]
Closed until further notice... see https://github.com/Chicken-Bones/NotEnoughItems/issues/176

**Comment by crunkazcanbe** [comment]
I want to thank you guyz! I know it was no big deal so i really appreciate what you guys are doing with one of my favorite mods. This and mekanism are the two best bods for minecraft . They go together so well . :-)

**Comment by crunkazcanbe** [comment]
Sorry , but after updating to the DEV build of This mod that fixes the gui tabs it is even worse now as you can see in the image below 

https://imgur.com/Hl0NOpF

**Comment by micdoodle8** [comment]
Can I get a forge log, from the client that you're able to recreate that problem in please? Also, do you have potions active during that screenshot?

**Comment by crunkazcanbe** [comment]
Yes sir there are potions active forgot to mention that. I will give you my whole forge log soon . There are no errors or crashes in the log though.

**Comment by radfast** [comment]
I can reproduce a similar-looking issue by these steps, with an active potion effect:
1.  disable NEI using the 'O' key
2.  open player inventory then the Galacticraft inventory tab
3.  while the Galacticraft tab is still open, press the 'O' key to re-enable NEI

I have tested whether a potion effect counting down to __0:00__ and then being removed can result in a similar effect: it does not (because the vanilla inventory remains in the same position).

In my test, the issue is that enabling or disabling NEI _while an inventory tab is open_ makes the whole inventory shift position if there is an active potion effect..

**Comment by crunkazcanbe** [comment]
Sorry wish i could post more info but my power supply went out on my PC . New one on the way . Will post more soon ,but like the person above said . That's how it happens .

**Comment by CodStyx** [comment]
im having the same issue can someone please repost the solution,  because i can't find it,  plus that i couldnot find any other posts about this problem,

**Comment by EzerArch** [comment]
1. Galacticraft version?
2. Installed other mods that add inventory tabs? (For example, CustomNPCs)

**Comment by CodStyx** [comment]
galacticraft 3.0.6.233
and i guess its battle gear 2 bullseye 1.0.6.3
and there is another tab that adds mask and heart canisters, idk whats that called

**Comment by dj3520** [comment]
If you go to where your minecraft install is located (usually in C:\Users\your username\Appdata\Roaming\.minecraft if using the default launcher on windows) you should find a "mods" folder. If not and you're using a custom launcher it's harder to tell (the one I use puts everything in my Documents folder) Every .jar file in the mods folder is a mod. 

>galacticraft 3.0.6.233

This version is from Nov 21, 2014 1:27 AM. You can imagine how many fixes have happened since then. If you're playing a modpack then the author needs to be alerted how old this version is.

**Comment by CodStyx** [comment]
well im using the atlauncher, its hermitcraft modsouce

**Comment by CodStyx** [comment]
andthe 3rd mod is tinkers construct mod

**Comment by dj3520** [comment]
You'll have to contact them if you want the mod in the modpack updated. [I believe I've found a relevant link.](https://www.atlauncher.com/pack/HermitcraftModsauce) I'm not sure how active they are if their Galacticraft version is that old, plus I feel the majority of people have moved to newer minecraft versions. This very page says the issue was resolved in a later build, so that will more than likely fix your issue.
